### ISSUE TITLE
[BUG] Deleting Liquid files breaks development

### ISSUE THREAD

**Issue by jzelaya-manduka** [hayes0724/shopify-packer#91]
**Describe the bug**
While `packer start` or `packer watch` is running, deleting a liquid file in the `src` folder causes development to break. The file in the `src` folder is deleted, but that same file in the `dist` folder still remains.

**To Reproduce**
1. packer start
2. create a liquid file in the src folder (e.g. src/snippets/product-card.liquid)
3. Delete the newly created file

**Expected behavior**
Deleting a liquid file under any subdirectory in the `src` folder should also delete the file in the `dist` folder and upload the change to Shopify.

**Screenshots**
![Screen Shot 2021-04-30 at 3 59 17 PM](https://user-images.githubusercontent.com/67347164/116761636-0740a900-a9cd-11eb-8b82-88d68f8530c0.png)

**System (please complete the following information):**
 - OS: Mac OSX 11.3
 - Node Version v12.22.1
 - NPM Version 6.14.12
 - Browser chrome 90.0.4430.93

**Packer (please complete the following information):**
 - Version v2.1.5

**Theme (please complete the following information):**
 - [Shopify Packer theme](https://github.com/hayes0724/packer-blank-theme) True
 - Version 1.0.13

**Comment by hayes0724** [comment]
@jzelaya-manduka This has been fixed in [v2.1.9](https://github.com/hayes0724/shopify-packer/releases/tag/v2.1.9)

**Comment by jzelaya-manduka** [comment]
@hayes0724 I've upgraded to v2.2.0 and I no longer see the error in the console and deleted files in the `dist` folder are now removed as intended. However after running `packer start` and then adding and deleting the file, the deleted file in the `dist` folder doesn't trigger an update to remove it from Shopify. Is this the expected behavior?

![Screen Shot 2021-05-03 at 10 00 00 AM](https://user-images.githubusercontent.com/67347164/116907405-5a497480-abf6-11eb-86c4-04f8ecd6000a.png)
